src/app/common/counter/service/counter-http.service.ts

```diff
@@ -19,7 +19,7 @@ export class CounterHttpService {
     return this.http.post(url, body, { observe: "response" });
   }
 
-  put(url: string, body: any) {
-    return this.http.put(url, body);
+  put(url: string) {
+    return this.http.put(url,null);
```

Comment:
Why null?

Reply:
My only PUT request is to cancel a booking, and it has no body.